android/app/src/main/kotlin/com/jabook/app/jabook/audio/AudioPlayerService.kt

```
@@ -880,6 +903,64 @@ public class AudioPlayerService : MediaLibraryService() {
+                        player.volume = if (prevLufs != null && newLufs != null) {
+                            (player.volume + gain).coerceIn(0f, 1f)
+                        } else {
+                            gain.coerceIn(0f, 1f)
+                        }
+                        LogUtils.i(
+                            "AudioPlayerService",
+                            "Book loudness compensation applied: gain=$gain, lufs=$newLufs, book=$newBookId",
+                        )
+                    }
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)